meson.build

```diff
@@ -1609,6 +1609,16 @@ have = get_option('sysupdate').require(
         error_message : 'fdisk and openssl required').allowed()
 conf.set10('ENABLE_SYSUPDATE', have)
 
+have2 = get_option('sysupdated')
+if have2 == 'enabled' and not have
+        error('sysupdated requires sysupdate to be enabled')
+elif have2 == 'auto'
+        have2 = have and conf.get('BUILD_MODE_DEVELOPER') == 1
+else
+        have2 = false
+endif
+conf.set10('ENABLE_SYSUPDATED', have2)
```

Comment:
At this point, `have2` is `'enabled'` or `false`. Is it expected? Does `conf.set10()` support a `string` ?

Reply:
I tried `enabled`, but ENABLE_SYSUPDATED is zero.